models/containerized_model/templates/prediction_service.py

```diff
@@ -35,7 +35,7 @@ def main():
     if model_path is not None:
         # Copy files from remote path to local
         default_model_path = os.path.normpath(os.path.join(CURRENT_PATH, '../model', os.path.basename(model_path)))
-        read_and_save_file(model_path, default_model_path)
+        read_and_save_model(model_path, default_model_path)
```

Comment:
We should probably change the function name to `read_and_save_file` for consitency since `prediction_service_h2o_mojo` has it named as `read_and_save_file`. Check here: https://github.com/CognitiveScale/cortex-certifai-examples/blob/master/models/containerized_model/templates/prediction_service_h2o_mojo.py#L36